### PR TITLE
Reintroduce realization job status

### LINKED ISSUE #1486: Reintroduce (sub)pixel realization statuses
**Describe the bug**
Realization statuses were axed in the refactor, this was a mistake

**To Reproduce**
Run any experiment, observe that the realization is represented by only two colors.

**Expected behavior**
It should be represented by `1 + len(realization_jobs)` colors.

### PR DESCRIPTION
**Pull request by jondequinor**
**Issue**
Resolves #1486

### CODE REVIEW

**Comment by sondreso** [comment]
Would it be possible to add a small black border between the queue status and the inner box? (Or an alternative is to have black boarders around all of the small job boxes, but this could cause problems i think) It's a little bit hard to separate the status of the queue and the jobs.  
image

File: ert_gui/simulation/view/realization.py [hunk 1]
```diff
@@ -1,3 +1,5 @@
+import random
```
Comment (sondreso) [inline ert_gui/simulation/view/realization.py:1]: Is this needed?

File: ert_gui/model/snapshot.py [hunk 2]
```diff
@@ -1,3 +1,4 @@
+from collections import defaultdict
```
Comment (sondreso) [inline ert_gui/model/snapshot.py:1]: Is this needed?

**Review by sondreso — COMMENTED** [review]
Looks good! Only nitpick comments 😄

File: ert_gui/model/snapshot.py (hunk 2 above)
Comment (jondequinor) [reply ert_gui/model/snapshot.py:1]: Nope!

**Review by sondreso — APPROVED** [review]
💯

**Pull request merged**